rack-dev-mark.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |gem|
   gem.executables = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   gem.require_paths = ['lib']
 
-  gem.required_ruby_version = ['>= 2.2', '< 2.6']
+  gem.required_ruby_version = ['>= 2.2', '< 2.7']
```

Comment:
Thank you for the update. But I am wondering why the upper limit is required. It's a little conservative and needs yearly update. So, how about changing like this?

``` ruby
gem.required_ruby_version = '>= 2.2'
# or
gem.required_ruby_version = ['>= 2.2', '< 3.0']
```

Reply:
You have a point!

Reply:
Removed upper limit 😃